File: bridges/get-calls-bridge/GetCallsBridge.cs
//Download the .net sdk from ap.bandwidth.com/docs/helper-libraries/net
using System;
using Bandwidth.Net;

public class Program
{
  //API credentials which can be found on your account page at https://catapult.inetwork.com/pages/login.jsf
  private const string UserId = "u-userID";  //{user_id}
  private const string Token = "t-token"; //{token}
  private const string Secret = "secret"; //{secret}

  public static void Main()
  {
    try
    {
      var client = new Client(UserId, Token, Secret);
      foreach (var call in client.Bridge.GetCalls("brg-65dhjbanfdofnqwrmbasiei"))
      {
        Console.WriteLine($"{call.Id}\t{call.From} -> {call.To}");
      }
    }
    catch (Exception ex)
    {
      Console.Error.WriteLine(ex.Message);
      Environment.ExitCode = 1;
    }
  }
}

File: bridges/get-calls-bridge/get-calls-bridge.js
//Download the node sdk from ap.bandwidth.com/docs/helper-libraries/node-js
//API credentials which can be found on your account page at https://catapult.inetwork.com/pages/login.jsf
var userId = 'userId';  //{user_id}
var token = 'token'; //{token}
var secret = 'secret'; //{secret}

var Bandwidth = require('node-bandwidth');

var client = new Bandwidth({
	userId: userId,
	apiToken: token,
	apiSecret: secret
});

//Promise
client.Bridge.getCalls('brg-65dhjbanfdofnqwrmbasiei')
.then(function (response) {
	console.log(response);
});

//Callback
client.Bridge.getCalls('brg-65dhjbanfdofnqwrmbasiei',
	function (err, response) {
		if(err) {
			console.log(err);
		}
		else {
			console.log(response);
		}
	});